scripts/install-windows-prereqs.ps1

```diff
@@ -10,7 +10,7 @@ Param(
     [string]$SevenZipURL = 'https://www.7-zip.org/a/7z1806-x64.msi',
     [string]$SevenZipHash = 'F00E1588ED54DDF633D8652EB89D0A8F95BD80CCCFC3EED362D81927BEC05AA5',
     [string]$VSBuildToolsURL = 'https://aka.ms/vs/15/release/vs_buildtools.exe',
-    [string]$VSBuildToolsHash = '7D5B0220670BA1C174F0AE1FF2CE87D65A508E66C321431FBD4751F478037E12',
```

Comment:
Maybe a note in the script itself as to why we don't check it's hash? Or better, could we cache the tool? It's weird to be dependent on tools from elsewhere on the internet, especially if it means we cannot verify their integrity. I'm just thinking back to the day of downtime due to an externally hosted driver...